tls-tutorial/doc/use_your_own_certificate.md

```diff
@@ -267,8 +267,8 @@ Let's look at the component of the configuration options.
 `keyUsge`
 : Defines what a client should accept the certificate for. If something is not listed here, the client _must_ (enforced by the `critical` modifier) refuse any attempts to use the key for it. Case in point, this server key may not sign certificates.
 
-: [Non-repudiation](#non-repudiation) in this context means a cryptographic method to make sure that neither sender nor recipient of information can deny to have processed that information.
-
+[Non-repudiation](#non-repudiation) 
+: in this context means a cryptographic method to make sure that neither sender nor recipient of information can deny to have processed that information.
```

Comment:
The colon ":" in next line seems superfluous. 
Would it make sense to format all configuration options into a bullet list? 
Proposal:
```suggestion
* `[extensions]` is a section header. This header can be used for find a set of options in the configuration file.
* `keyUsge` defines what a client should accept the certificate for. If something is not listed here, the client _must_ (enforced by the `critical` modifier) refuse any attempts to use the key for it. Case in point, this server key may not sign certificates.
* [Non-repudiation](#non-repudiation) in this context means a cryptographic method to make sure that neither sender nor recipient of information can deny to have processed that information. The key can also be used for creating a digital signature, to encipher other keys, and as key agreement. If you are asking yourself why payload data encryption is not mentioned, remember that TLS uses asymmetric cryptography only until the symmetric session key is exchanged. Actual data payload is encrypted symmetrically.
* `basicConstraints` is explicitly added as a constraint here that this configuration is not CA certificates. You could drop that option if you wanted, because this is the default for certificates. 
```

BTW: The first sentence in the last bullet point I don't understand. First of all due to a potential grammar issue: this configuration [singular) is not CA certificates [plural]. Maybe a "for" is missing?